runtime/oti/VMHelpers.hpp

```diff
@@ -2012,6 +2013,20 @@ class VM_VMHelpers
 		return J9_ARE_NO_BITS_SET(omrthread_get_category(vmThread->osThread), nonJavaThreads);
 	}
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+	/*
+	 * Queries whether the JVM is running in single thread mode
+	 *
+	 * @param[in] vm Java VM token
+	 * @return True if JVM is in single thread mode, false otherwise
+	 */
+	static VMINLINE bool
+	isJVMInSingleThreadMode(J9JavaVM *vm)
```

Comment:
why is this moved?

Reply:
Restored, was thinking to take out `CRIUHelpers.hpp`, apparently isn't right direction.